### ISSUE TITLE
Webapp : Few of modals dont have a11y classes

### ISSUE THREAD

**Issue by M-ZubairAhmed** [mattermost/mattermost-server#13865]
#### Summary
Few of the modals and popover don't have respective a11y classes as others.

#### Steps to reproduce
- open search from top right
- inspect class name of the search drop down
- it doesn't contain a11y popup class
- similarly for other modals and popovers too

#### Expected behavior
They should have respective popovers.
For modal : `a11y__modal`
For popover/popups : `a11y__popup`

#### Possible fixes
This has been discussed in [community chat](https://community.mattermost.com/core/pl/wkp8jns9ajnxzpjzkywn7ysxyc).
- Relevant classes will be added to modals and popups
- Will be tested for a11y

**Comment by esethna** [comment]
Thanks @M-ZubairAhmed is there any expected change in terms of how this will affect screen reader usability?

**Comment by esethna** [comment]
@deanwhillier or @devinbinnie can we please create a JIRA ticket for these changes?
